code/modules/hydroponics/grown/towercap.dm

```diff
@@ -149,6 +160,11 @@
 	StartBurning()
 
 /obj/structure/bonfire/attackby(obj/item/W, mob/user, params)
+	if(istype(W, /obj/item/stack/sheet/wood))
```

Comment:
You can't use raw towercap logs to fuel the bonfire made of raw towercap logs?

Reply:
Yup.

Reply:
Added looooog to the fuel sources